Source/IGListGridCollectionViewLayout.h

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <UIKit/UIKit.h>
+
+/**
+ This layout only works with section controllers that return 1 item.
+ */
+@interface IGListGridCollectionViewLayout : UICollectionViewLayout
+
+/**
+ The scroll direction of the grid.
+ */
+@property (nonatomic, assign) UICollectionViewScrollDirection scrollDirection;
```

Comment:
Not sure about this. I plan to support changing scroll direction in the future. I saw UICollectionViewFlowLayout allows us to change direction after init.